org.civicrm.reqangular/test/directives/horizontal-scroll-shadows.directive.spec.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angular',
+  'common/lodash',
+  'common/angularMocks',
+  'common/directives/horizontal-scroll-shadows.directive'
+], function (angular, _) {
+  'use strict';
+
+  describe('horizontalScrollShadows directive', function () {
```

Comment:
you don't put the type of the thing you are testing, just the name

Reply:
Done.